.travis.yml

```diff
@@ -38,3 +39,10 @@ jobs:
           env: LINTER=pydocstyle
           script: make pydocstyle
           if: branch IN (master, dev)
+        - stage: docs build
+          python: 3.5
```

Comment:
Any reason for v3.5, rather than say 3.6?

Reply:
It's the same version as is specified for linting... which is that version because it's the version that Travis has pre-installed, so it means the tests run marginally faster.

Reply:
Ah yes, I see the linters now...